src/core/engines/Cesium/Feature/Tileset/index.tsx

```diff
@@ -43,17 +207,25 @@ export default function Tileset({
 
   const tilesetUrl = useMemo(() => {
     return sourceType === "osm" && isVisible
-      ? IonResource.fromAssetId(96188) // https://github.com/CesiumGS/cesium/blob/1.69/Source/Scene/createOsmBuildings.js#L50
+      ? IonResource.fromAssetId(1240402) // https://github.com/CesiumGS/cesium/blob/1.69/Source/Scene/createOsmBuildings.js#L50
       : isVisible
       ? tileset
       : null;
   }, [isVisible, sourceType, tileset]);
```

Comment:
Too long hooks, so they can be separated into `hooks.ts`

Reply:
I've moved logic to hooks.
[refactor: move logic to hooks in tileset component](https://github.com/reearth/reearth-web/pull/377/commits/a85b7bd5b391dbb1676f96d389fbb60f542738b8)